rust-connectors/models/fluvio-model-sql/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+use serde::Deserialize;
+use serde::Serialize;
+
+/// Top-level list of supported operations in the SQL model.
+#[derive(Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub enum Operation {
+    Insert { table: String, values: Vec<Value> },
+}
+
+/// Value with SQL column name and supported SQL type.
+#[derive(Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct Value {
+    pub column: String,
+    pub raw_value: String,
+    pub r#type: Type,
```

Comment:
There's nothing wrong with using `r#type` as a field or varable name. I personally prefer to use `type_` with [`#[serde(rename = "type"))]`](https://serde.rs/field-attrs.html#rename). It just makes it easier to _type_.

Reply:
Yeah, I agree. Done.